### ISSUE TITLE
PHP 

### ISSUE THREAD

**Issue by akahige** [dflydev/dflydev-doctrine-orm-service-provider#56]

**Comment by marcojanssen** [comment]
Or raise the php version required and tag it, why should this stay compatible with a EOL php version

**Comment by akahige** [comment]
I agree that new versions can be 5.4+ compatible, but the problem is that none of the available tags are 5.3+ compatible.
